includes/admin/class-insert-term.php

```diff
@@ -0,0 +1,53 @@
+<?php
+namespace bedIQ\Admin;
+/**
+ * Class Insert_Term
+ */
+class Insert_Term {
+    /**
+     * Constructor for Insert_Term
+     */
+    function __construct() {
+        add_action( 'init', array( $this, 'create_new_term' ), 9999 );
```

Comment:
You are attempting to create terms on init, on every request. Rather do it on plugin activation hook.